app/views/user_mailer/featured_circuit_email.html.erb

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
Can we use email frameworks like zurb(https://foundation.zurb.com/emails.html). Because it will be hard to maintain a file with inline css

Reply:
Will it be okay if I work on this one is separate PR? All the templates will be changed in that one.